### PR TITLE
Eds 34 leaflet

### PR DESCRIPTION
**Pull request by joncooper65**
I could really do with this adding to CEH view, but I still need to add it to the elements documentation

### CODE REVIEW

File: src/elements/select.css [hunk 1]
```diff
@@ -31,6 +31,13 @@
     color: #4a4a4a;
     border-bottom: solid 1px #4a4a4a;
     transition: border 0.2s ease-in-out;
+        &:hover {
+        cursor: pointer;
+    }
+}
+
+.select-outer-container.is-hide {
```
Comment (lukejpreston) [inline src/elements/select.css]: `is-hidden`?

File: src/elements/select.js [hunk 2]
```diff
@@ -216,7 +217,10 @@ class Select extends Component {
     if (this.state.filled) className += ' is-filled'
     if (this.props.className) className += ` ${this.props.className}`
 
-    return <div className='select-outer-container'>
+    let outerContainerClassName = `select-outer-container`
+    if (this.props.visibility && (this.props.visibility === 'hidden')) outerContainerClassName += ' is-hide'
```
Comment (lukejpreston) [inline src/elements/select.js]: you could change this to `outerContainerClassName += `is-${this.props.visbility}` `
then after the class add `Select.defualtProps = {visibility: 'visible'}`

Comment (lukejpreston) [reply src/elements/select.js]: there is actually a bulma helper for this <https://bulma.io/documentation/modifiers/helpers/> `is-invisible` I don't know if it is available in our version of bulma but I think it is

File: src/elements/select.css (hunk 1 above)
Comment (joncooper65) [reply src/elements/select.css]: changed to bulma's is-invisible

File: src/elements/select.js (hunk 2 above)
Comment (joncooper65) [reply src/elements/select.js]: now its as you recommended - much simpler thanks

Comment (joncooper65) [reply src/elements/select.js]: that worked - and simplified it as you suggested

**Comment by lukejpreston** [comment]
@joncooper65 looking good, you have some merge conflicts. As far as the `dist/index.js` is concerned, once you have resolved the conflicts re-run `npm run dist` and then commit that change

**Pull request merged**